sci-libs/coinasl/coinasl-2.0.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Ampl Solver Library (ASL)"
+HOMEPAGE="https://github.com/coin-or-tools/ThirdParty-ASL"
+SOLVERS_SHA="64919f75f"
+SRC_URI="https://coin-or-tools.github.io/ThirdParty-ASL/solvers-${SOLVERS_SHA}.tgz
+	https://github.com/coin-or-tools/ThirdParty-ASL/archive/refs/tags/releases/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/ThirdParty-ASL-releases-${PV}"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+PATCHES=(
+	"${FILESDIR}/${P}-getrusage.patch"
+	"${FILESDIR}/${P}-dtoa.patch"
+)
```

Comment:
Why are you not contributing these patches upstream? Depending on mysterious undocumented commits of BuildTools to rebuild their autoconf scripts is a really bad situation especially considering it may not actually work with recent autoconf versions depending on which coin-or package we are talking about, which BuildTools version, and how recently they released a major version.

It's the opposite of futureproof.

Reply:
I will submit an MR for the newest patch. The old patch is already present in upstream but for some reason only as a patch file.

Reply:
https://github.com/coin-or-tools/ThirdParty-ASL/pull/7